modules/tinymce/changelog.txt

```diff
@@ -1,3 +1,5 @@
+Version 5.7.1 (TBD)
+    Fixed context menu items with names that contained uppercase characters were not displayed #TINY-7072
```

Comment:
```suggestion
    Fixed a bug where context menu items with names that contained uppercase characters were not displayed #TINY-7072
```

Something like this? I want to say "were not **being** displayed" but seeing as that's not allowed, I feel like this is the next best thing